ecommerce/coupons/views.py

```diff
@@ -135,6 +135,8 @@ def get_context_data(self, **kwargs):
             })
             self.template_name = 'coupons/offer.html'
 
+        context_data['code_name'] = voucher.name
```

Comment:
Why not return the entire object, so that in the future we can use the other fields and avoid making other changes... @Jacatove